.gitignore

```
@@ -97,3 +97,6 @@ postcss_error.log
 # ripgrep ignore file
 .rgignore
 config/definitions.rb
+
+# ignore redis backup
+dump.rdb
```

Not sure if we want to add this newline to make GH happy 😃

---

it will be added by most editors when you open the file anyway, so let's add it :D

```suggestion
dump.rdb

```